test/test.js

```diff
@@ -364,6 +364,28 @@ describe(`rendering`, function() {
           );
         });
       });
+
+      describe(`mixins`, function() {
```

Comment:
Please include these two test cases under the existing "multiple files" context rather than making a new one

Reply:
eh, never mind me - no reason to have these single-file mixin tests under "multiple files"